Classes/KIFTextInputTraitsOverrides.h

```diff
@@ -0,0 +1,12 @@
+//
+//  KIFTextInputTraitsOverrides.h
+//  KIF
+//
+//  Created by Harley Cooper on 1/31/18.
+//
+
+#import <Foundation/Foundation.h>
+
+FOUNDATION_EXTERN void KIFSetAutocorrect(BOOL);
```

Comment:
NIT: Would be slightly more clear as `KIFAllowDefaultAutocorrectBehavior(BOOL)` and same for the two below